Testscripts/Windows/VALIDATE-DA.ps1

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+<#
+.Synopsis
+    This script validates Dependency Agent.
+
+.Description
+    The script will install the Latest Dependency Agent and validate install/uninstall
+    scenarios as well as enable/disable scenarios.
+'
+#>
+param([String] $TestParams,
+      [object] $AllVmData)
+function Main {
+    param (
+        $VMName,
+        $Ipv4,
+        $VMPort,
+        $VMUserName,
+        $VMPassword
+    )
+
+    $remoteScript = "validate-da.sh"
+    #######################################################################
+    #
+    #	Main body script
+    #
+    #######################################################################
+
+    # Checking the input arguments
+    if (-not $VMName) {
```

Comment:
Insufficient validation: As written `Main($true, "1.2.3.4", 22, "user", "password")` will get past this check.